rust/property-defs-rs/tests/test_migrations/20240830124836_setup_propdefs_tables.sql

```diff
@@ -9,6 +9,7 @@ CREATE TABLE IF NOT EXISTS posthog_eventdefinition (
     volume_30_day INTEGER,
     query_usage_30_day INTEGER,
     team_id INTEGER NOT NULL,
+    project_id INTEGER NULL,
```

Comment:
Better to keep these in sync and make them a bigint

Reply:
Good point, now i64!